app/_locales/pt_BR/messages.json

```diff
@@ -128,44 +203,53 @@
     "message": "O saldo pode estar desatualizado"
   },
   "basic": {
-    "message": "Básicas"
+    "message": "Básico"
+  },
+  "blockExplorerUrl": {
+    "message": "Block Explorer"
   },
   "blockExplorerView": {
-    "message": "Ver conta em $1",
+    "message": "Exibir conta em $1",
     "description": "$1 replaced by URL for custom block explorer"
   },
   "blockiesIdenticon": {
-    "message": "Usar o Identicon de Blocos"
+    "message": "Usar identicon de Blockies"
   },
   "browserNotSupported": {
     "message": "Seu navegador não é compatível..."
   },
   "builtInCalifornia": {
-    "message": "O MetaMask foi projetado e desenvolvido na Califórnia."
+    "message": "O MetaMask é projetado e construído na Califórnia."
```

Comment:
When using `built` referring to software development, I believe we use `desenvolvido` instead of `construido`.

Construido is more when we are refering to a building or some object that was built.

I'd go with:
```
O MetaMast é projetado e desenvolvido na Califórnia
```

Reply:
Makes sense!
```suggestion
    "message": "O MetaMask é projetado e desenvolvido na Califórnia."
```